npm/ipsm/ipsm.go

```diff
@@ -613,57 +605,59 @@ func (ipsMgr *IpsetManager) DestroyNpmIpsets() error {
 		return nil
 	}
 
-	entry := &ipsEntry{
-		operationFlag: util.IpsetFlushFlag,
-	}
-
+	destroyFailureCount := 0
 	for _, ipsetName := range ipsetLists {
-		entry := &ipsEntry{
+		flushEntry := &ipsEntry{
 			operationFlag: util.IpsetFlushFlag,
 			set:           ipsetName,
 		}
+		_, flushError := ipsMgr.run(flushEntry)
 
-		if _, err := ipsMgr.run(entry); err != nil {
-			metrics.SendErrorLogAndMetric(util.IpsmID, "{DestroyNpmIpsets} Error: failed to flush ipset %s", ipsetName)
+		deleteEntry := &ipsEntry{
```

Comment:
Can we destroy ipset even when flushing entries is failed? 
As far as I understand if the ipset is refer to others, we cannot destroy ipset or flush entries.
If so, we do not need to walk through the rest of code and do `continue`

Reply:
not 100% sure. Thoughts? @matmerr @vakalapa

Reply:
If flush failed, there is a high chance for destroy to fail too.

Reply:
I'll leave this code just in case there's a chance you can get an error when flushing and still successfully delete